server/handlers/instrumented_project_command_output_handler.go

```diff
@@ -1,44 +1,33 @@
 package handlers
 
 import (
+	"fmt"
+
 	stats "github.com/lyft/gostats"
-	"github.com/runatlantis/atlantis/server/events/models"
+	"github.com/runatlantis/atlantis/server/logging"
 )
 
 type InstrumentedProjectCommandOutputHandler struct {
 	ProjectCommandOutputHandler
-	numChans stats.Gauge
+	numWSConnnections stats.Gauge
+	logger            logging.SimpleLogging
 }
 
-func NewInstrumentedProjectCommandOutputHandler(prjCmdOutputHandler ProjectCommandOutputHandler, statsScope stats.Scope) ProjectCommandOutputHandler {
+func NewInstrumentedProjectCommandOutputHandler(prjCmdOutputHandler ProjectCommandOutputHandler, statsScope stats.Scope, logger logging.SimpleLogging) ProjectCommandOutputHandler {
 	return &InstrumentedProjectCommandOutputHandler{
 		ProjectCommandOutputHandler: prjCmdOutputHandler,
-		numChans:                    statsScope.Scope("log_streaming").NewGauge("num_ws_chans"),
+		numWSConnnections:           statsScope.Scope("job").Scope("websocket").NewGauge("connections"),
+		logger:                      logger,
 	}
 }
 
-func (p *InstrumentedProjectCommandOutputHandler) Clear(ctx models.ProjectCommandContext) {
-	p.ProjectCommandOutputHandler.Clear(ctx)
-}
-
-func (p *InstrumentedProjectCommandOutputHandler) Send(ctx models.ProjectCommandContext, msg string) {
-	p.ProjectCommandOutputHandler.Send(ctx, msg)
-}
-
 func (p *InstrumentedProjectCommandOutputHandler) Receive(projectInfo string, receiver chan string, callback func(msg string) error) error {
-	p.numChans.Inc()
-	defer p.numChans.Dec()
+	p.numWSConnnections.Inc()
+	defer func() {
+		// Log message to ensure numWSConnnections gauge is being updated properly.
+		// TODO: Remove when removing the feature flag for log streaming.
```

Comment:
Mind creating a ticket for feature flag removal and then adding the ID here so we can easliy search and replace these things?

Reply:
Done!